plotly/tools.py

```diff
@@ -1670,6 +1722,321 @@ def create_streamline(x, y, u, v,
+            for more information on valid kwargs call
+            help(plotly.graph_objs.Scatter)
+
+        :rtype (trace): returns data for ohlc increasing units or decreasing
+            units.
+
+        Example 1: Plot ohlc chart
+        ```
+        import plotly.plotly as py
+        import plotly.tools as tls
+        from plotly.graph_objs import *
+
+        # Add data
+        open_data = [33.0, 33.3, 33.5, 33.0, 34.1]
+        high_data = [33.1, 33.3, 33.6, 33.2, 34.8]
+        low_data = [32.7, 32.7, 32.8, 32.6, 32.8]
+        close_data = [33.0, 32.9, 33.3, 33.1, 33.1]
+
+        # Create ohlc increasing units
+        ohlc_increase = tls.TraceFactory.create_ohlc(open_data, high_data,
+                                                     low_data, close_data,
+                                                     direction='increasing')
+
+        # Create ohlc decreasing units
+        ohlc_decrease = tls.TraceFactory.create_ohlc(open_data, high_data,
+                                                     low_data, close_data,
+                                                     direction='decreasing')
+
+        # Plot
+        fig = Figure()
+        fig['data'].append(ohlc_increase)
+        fig['data'].append(ohlc_decrease)
+        url = py.plot(fig, filename='ohlc')
+        ```
+
+        Example 2: Plot ohlc chart with date labels
+        ```
+        import plotly.plotly as py
+        import plotly.tools as tls
+        from plotly.graph_objs import *
+
+        from datetime import datetime
+
+        # Add data
+        open_data = [33.0, 33.3, 33.5, 33.0, 34.1]
+        high_data = [33.1, 33.3, 33.6, 33.2, 34.8]
+        low_data = [32.7, 32.7, 32.8, 32.6, 32.8]
+        close_data = [33.0, 32.9, 33.3, 33.1, 33.1]
+        dates = [datetime(year=2013, month=10, day=10),
+                 datetime(year=2013, month=11, day=10),
+                 datetime(year=2013, month=12, day=10),
+                 datetime(year=2014, month=1, day=10),
+                 datetime(year=2015, month=2, day=10)]
+
+        # Create ohlc trace of increasing units
+        ohlc_increase = tls.TraceFactory.create_ohlc(open_data, high_data,
+                                                     low_data, close_data,
+                                                     direction='increasing')
+
+        # Create ohlc trace of decreasing units
+        ohlc_decrease = tls.TraceFactory.create_ohlc(open_data, high_data,
+                                                     low_data, close_data,
+                                                     direction='decreasing')
+
+        # Create layout with dates as x-axis labels
+        fig = dict(data=[ohlc_increase, ohlc_decrease],
+              layout=dict(xaxis = dict(ticktext = dates,
+                                       tickvals = [1, 2, 3, 4, 5 ])))
+
+        # Plot
+        url = py.plot(fig, filename='ohlcs_dates', validate=False)
+        ```
+        """
```

Comment:
it'd be cool to have a pandas example in here too:

```
import pandas.io.data as web

start = datetime.datetime(2010, 1, 1)
end = datetime.datetime(2013, 1, 27)
df = web.DataReader("aapl", 'yahoo', start, end)
py.iplot([tls.TraceFactory.create_ohlc(df.Open, df.High, df.Low, df.Close)])
```
![ohlc-aapl](https://cloud.githubusercontent.com/assets/1280389/8843947/5e71e154-30dd-11e5-9c26-38788584af60.png)